projects/cdk/services/pan.service.ts

```diff
@@ -21,13 +21,13 @@ export class TuiPanService extends Observable<readonly [number, number]> {
                 .pipe(
                     switchMapTo(
                         merge(
-                            typedFromEvent(nativeElement, 'touchmove', {
+                            typedFromEvent(documentRef, 'touchmove', {
                                 passive: true,
                             }).pipe(
                                 filter(({touches}) => touches.length < 2),
                                 map(({touches}) => touches[0]),
                             ),
-                            typedFromEvent(nativeElement, 'mousemove'),
```

Comment:
@vladimirpotekhin 
is it okay and doesnt break anything ?

(these changes are required because if we drag something and cursor leaves original point, it stops listening `*move` events)

Reply:
Yes, I think it's ok 👍